demosplan/DemosPlanCoreBundle/Resources/config/config_dev_container.yml

```diff
@@ -40,7 +40,6 @@ parameters:
         - '*.demos-europe.eu'
         - 'cdnjs.cloudflare.com'
 
-    rabbitmq_host: "mq"
```

Comment:
Shouldn't we set the `rabbitmq_dsn` parameter for the container setup in this file now?

Reply:
No, we cannot, as this file is checked in into the public git

Reply:
That reasoning doesn't exactly match with database, elasticsearch and mailhog being configured here too. 🤔

Reply:
But they are configured without users an passwords. Rabbitmq dns would contain both

Reply:
the variable `rabbitmq_host` does not exist any more

Reply:
Well. We need another seamless solution soon then I guess.